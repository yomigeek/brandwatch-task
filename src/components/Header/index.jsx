import styled from "styled-components";

const HeaderBox = styled.header`
  height: 80px;
  width: 100%;
  background: #fff1dc;
  text-align: center;
  color: #fff;
`;

const Header = () => {
  return (
    <HeaderBox>
      <a href="/">
        <img
          src="https://res.cloudinary.com/kugoo/image/upload/v1645483046/upload/logo.png"
          alt="logo"
          height="80px"
          width="80px"
        />
      </a>
    </HeaderBox>
  );
};

export default Header;
