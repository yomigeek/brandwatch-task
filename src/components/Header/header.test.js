/* eslint-disable no-undef */
import { render } from '@testing-library/react'
import Header from './index'

test('renders component snapshot', () => {
    render(<Header />)
    const asFragment = render(<Header />).asFragment
    expect(asFragment()).toMatchSnapshot()
})
