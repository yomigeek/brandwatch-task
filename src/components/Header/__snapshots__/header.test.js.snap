// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders component snapshot 1`] = `
<DocumentFragment>
  <header
    class="sc-bdvvtL eEUwUe"
  >
    <a
      href="/"
    >
      <img
        alt="logo"
        height="80px"
        src="https://res.cloudinary.com/kugoo/image/upload/v1645483046/upload/logo.png"
        width="80px"
      />
    </a>
  </header>
</DocumentFragment>
`;
